test/Test/Toml/Gen.hs

```diff
@@ -150,9 +152,41 @@ genZoned = do
     pure $ ZonedTime local zTime
 
 genDate :: MonadGen m => m DateTime
-genDate = Gen.choice 
+genDate = Gen.choice
     [ Day   <$> genDay
     , Hours <$> genHours
     , Local <$> genLocal
     , Zoned <$> genZoned
-    ]
\ No newline at end of file
+    ]
+
+genBool :: MonadGen m => m Bool
+genBool = Gen.bool
+
+genInteger :: MonadGen m => m Integer
+genInteger = toInteger <$> Gen.int (Range.constantBounded @Int)
+
+genDouble :: MonadGen m => m Double
+genDouble = Gen.double $ Range.constant @Double (-1000000.0) 1000000.0
+
+genText :: MonadGen m => m Text
+genText = Gen.text (Range.constant 0 256) Gen.alphaNum
+
+noneArrayList :: MonadGen m => [m AnyValue]
+noneArrayList =
+    [ AnyValue . Bool    <$> genBool
+    , AnyValue . Integer <$> genInteger
+    , AnyValue . Double  <$> genDouble
+    , AnyValue . Text    <$> genText
+    , AnyValue . Date    <$> genDate
+    ]
+
+genArrayFrom :: MonadGen m => m AnyValue -> m (Value 'TArray)
+genArrayFrom noneArray = fromMaybe (error "Error in genArrayFrom") . toMArray <$> Gen.list (Range.constant 3 6) noneArray
+
+genArray :: MonadGen m => m (Value 'TArray)
+genArray = Gen.recursive Gen.choice
+    [Gen.choice $ map genArrayFrom noneArrayList]
+    [genArray]
```

Comment:
It would be great to show results of `genArray` generator. It's not obvious from code that it can generate arrays of arrays. But it should. I think it's not enough to have only `[genArray]` here. I guess the last argument of `recursive` should be something smarter.

Reply:
Where to show results, here or in code comments?

Reply:
Just here. No need to put under comments. Just want to make sure that everything works as expected.